Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -59,6 +59,8 @@ export default class ReportsView extends Vue {
     private reportType = "";
     private startDate?: Date | null = null;
     private endDate?: Date | null = null;
+    private selectedStartDate?: Date | null = null;
```

Comment:
looks like the value is never actually undefined. Can the question mark be removed?
```private selectedStartDate: Date | null = null;```
same for the other variables